SXR/SDK/backend_daydream/src/main/AndroidManifest.xml

```diff
@@ -17,7 +17,7 @@
 
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="org.gearvrf.daydream"
```

Comment:
aren't there daydream specific manifest flags that are required?

Reply:
This is what we have in the daydream manifest:
```
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    package="org.gearvrf.daydream"
    android:versionCode="1"
    android:versionName="1.0" >

    <!-- copied from https://github.com/googlevr/gvr-android-sdk/blob/master/samples/sdk-treasurehunt/src/main/AndroidManifest.xml-->

    <!-- Required for vibration feedback when the trigger action is performed. -->
    <uses-permission android:name="android.permission.VIBRATE"/>
    <!-- Required to read the paired viewer's distortion parameters. -->
    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
    <!-- Required to detect viewer insertion. -->
    <uses-permission android:name="android.permission.NFC"/>

    <!-- Make accelerometer and gyroscope hard requirements for good head tracking. -->
    <uses-feature android:name="android.hardware.sensor.accelerometer" android:required="true"/>
    <uses-feature android:name="android.hardware.sensor.gyroscope" android:required="true"/>

    <!-- Indicates use of Android's VR-mode, available only on Android N+. -->
    <uses-feature android:name="android.software.vr.mode" android:required="false"/>
    <!-- Indicates use of VR features that are available only on Daydream-ready devices. -->
    <uses-feature android:name="android.hardware.vr.high_performance" android:required="false"/>

    <!-- done copying from https://github.com/googlevr/gvr-android-sdk/blob/master/samples/sdk-treasurehunt/src/main/AndroidManifest.xml-->

</manifest>
```

The have other requirements but they apply to the activity so manifest merging won't help there.